src/bundle/Resources/views/admin/content_type/list.html.twig

```diff
@@ -8,27 +8,38 @@
     <div class="ez-table-header">
         <div class="ez-table-header__headline">{{ 'content_type.view.list.title'|trans({ '%identifier%': content_type_group.identifier })|desc('Content Types in %identifier%') }}</div>
         <div>
-            <a href="{{ path('ezplatform.content_type.add', {contentTypeGroupId: content_type_group.id}) }}"
-               title="{{ 'content_type.view.list.action.add'|trans|desc('Create a Content Type') }}"
-               class="btn btn-primary">
-                <svg class="ez-icon ez-icon-create">
-                    <use xmlns:xlink="http://www.w3.org/1999/xlink"
-                         xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#create"></use>
-                </svg>
-            </a>
-            {% set modal_data_target = 'delete-content-types-modal' %}
-            <button id="delete-content-types" type="button" class="btn btn-danger" disabled data-toggle="modal"
-                    data-target="#{{ modal_data_target }}" title="{{ 'content_type.view.list.action.delete'|trans|desc('Delete Content Type') }}">
-                <svg class="ez-icon ez-icon-trash">
-                    <use xmlns:xlink="http://www.w3.org/1999/xlink"
-                         xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#trash"></use>
-                </svg>
-            </button>
-            {% include '@ezdesign/admin/bulk_delete_confirmation_modal.html.twig' with {
-                'id': modal_data_target,
-                'message': 'content_type.modal.message'|trans|desc('Do you want to delete Content Type?'),
-                'data_click': '#content_types_delete_delete',
-            }%}
+            {% if can_create %}
+                <a
+                    href="{{ path('ezplatform.content_type.add', {contentTypeGroupId: content_type_group.id}) }}"
+                    title="{{ 'content_type.view.list.action.add'|trans|desc('Create a Content Type') }}"
+                    class="btn btn-primary">
+                    <svg class="ez-icon ez-icon-create">
+                        <use xmlns:xlink="http://www.w3.org/1999/xlink"
```

Comment:
This part: `xmlns:xlink="http://www.w3.org/1999/xlink"` is not needed at all.

Reply:
Please, fix it in other places as well

Reply:
@sunpietro fixed. As we discussed I will make another PR for other places.